frontend/dispatcher.lua

```diff
@@ -206,6 +207,7 @@ local dispatcher_menu_order = {
     "back",
     "previous_location",
     "latest_bookmark",
+    "panel_zoom_toggle",
```

Comment:
this should probably go before or after
```lua
    "cycle_highlight_action",
    "cycle_highlight_style",
```

if you put it after then please remove the separator from `cycle_highlight_style` and add it to `panel_zoom_toggle`